spec/features/merchant/merchant_invoices_index_spec.rb

```diff
@@ -31,4 +32,34 @@
     end
   end
   # User Story 14 Testing End
+
+  # User Story 15 Testing Begins
+
+  # As a merchant
+  # When I visit my merchant's invoice show page (/merchants/:merchant_id/invoices/:invoice_id)
+  # Then I see information related to that invoice including:
+
+  # Invoice id
+  # Invoice status
+  # Invoice created_at date in the format "Monday, July 18, 2019"
+  # Customer first and last name
+
+  it 'displays the invoice id and status on the merchants show page' do
+    visit merchant_invoice_path(@merchant1, @invoice1)
+
+    expect(page).to have_content(@invoice1.id)
+    expect(page).to have_content(@invoice1.status)
+  end
+
+  it 'displays the invoice created_at date in the format "Monday, July 18, 2019"' do
+    visit merchant_invoice_path(@merchant1, @invoice1)
+
+    expect(page).to have_content(@invoice1.created_at.strftime('%A, %B %d, %Y'))
```

Comment:
Looks good! If you want to make some of your tests more robust in the future, you can add a capybara clause of `count: 1` when something should only appear once on the page. so this could look like:

`expect(page).to have_content(@invoice1.created_at.strftime('%A, %B %d, %Y'), count: 1)`